_layouts/profiles.html

```diff
@@ -0,0 +1,53 @@
+---
+layout: default
+---
+
+<!-- about.html -->
+      <div class="post">
+        <header class="post-header">
+          <h1 class="post-title">
+           {% if site.title == "blank" -%}<span class="font-weight-bold">{{ site.first_name }}</span> {{ site.middle_name }} {{ site.last_name }}{%- else -%}{{ site.title }}{%- endif %}
+          </h1>
+          <p class="desc">{{ page.subtitle }}</p>
+        </header>
```

Comment:
the page header of the profiles layout should be different from how it is displayed on the about page. ideally, we want the title and description to be specified in the front matter of the page explicitly instead of using `site.first_name`, `site.middle_name`, etc.

this is how it looks right now (`You R. Name` is irrelevant if the page contains information about other people):
![Screenshot 2023-06-25 at 6 23 27 AM](https://github.com/alshedivat/al-folio/assets/2126561/03a5a7b0-7070-4c19-8d01-40f165092807)